migrations/sql/V2019.04.12.13.45__create_address_table.sql

```diff
@@ -0,0 +1,86 @@
+CREATE TABLE IF NOT EXISTS address (
+    address_id        integer                   PRIMARY KEY,
+    party_guid        uuid                      NOT NULL   ,
+    suite_no          varchar(5)                           ,
+    address_line_1    varchar(50)                          ,
+    address_line_2    varchar(50)                          ,
+    city              varchar(50)                          ,
+    sub_division_code varchar(2)                           ,
+    post_code         varchar(6)                           ,
+    address_type_code varchar(3)  DEFAULT 'CAN' NOT NULL
+);
+
+ALTER TABLE address OWNER TO mds;
+
+CREATE SEQUENCE address_id_seq
+    START WITH 1
+    INCREMENT BY 1
+    NO MINVALUE
+    NO MAXVALUE
+    CACHE 1;
```

Comment:
_serial_ as the column type is a short hand for exactly this. 

`    address_id        serial   PRIMARY KEY,` will handle the sequence.

Reply:
Yeah, I think I saw that in one of your other PRs. I just copied this format from our big, initial migration file. If we'd like to use serial instead in future migrations, I can update this one so that it can be used as a proper point of reference. Otherwise, I don't think it should really matter

Reply:
I don't think it matters, that base schema was generated by pg_dump, so it is as verbose as it can be. the name of the sequence has to be perfect to it to play along with sqlalchemy (I think), but you have named it properly.